packages/medusa-payment-stripe/src/services/stripe-provider.js

```diff
@@ -141,6 +125,7 @@ class StripeProviderService extends PaymentService {
     const amount = await this.totalsService_.getTotal(cart)
 
     const intentRequest = {
+      description: cart?.context?.description ?? this.options?.description,
```

Comment:
Would argue, that plugin options should take precedence over cart context

Reply:
I wasn't sure, i was thinking that if you want to out a special description depending on the provenance of the order etc if the configuration is taking precedence it means that you can't. Wdty? The description could integrate some interesting information about the context of the payment

Reply:
@olivermrbl wdty?